geth/jail/handlers.go

```diff
@@ -66,23 +66,54 @@ func registerHandlers(jail *Jail, cell common.JailCell, chatID string) error {
 	return nil
 }
 
+// makeAsyncSendHandler returns jeth.sendAsync() handler.
+func makeAsyncSendHandler(jail *Jail, cellInt common.JailCell) func(call otto.FunctionCall) otto.Value {
+	// FIXME(tiabc): Get rid of this.
+	cell := cellInt.(*Cell)
+	return func(call otto.FunctionCall) otto.Value {
+		go func() {
+			response := jail.Send(call, cell.VM)
+
+			if fn := call.Argument(1); fn.Class() == "Function" {
```

Comment:
Hope it's always a a case `params, callback` :)

More bulletproof solution could be: `fn := call.Argument(call.NumberOfArguments - 1)`, provided there is a prop which returns a number of arguments.

Reply:
Nope, `send` in web3 always has a single argument. `sendAsync` always has only 2 arguments. If it changes in web3 we'd better receive an error during tests rather than come across some non-obvious behavior.